### ISSUE TITLE
Internal error for similar requests

### ISSUE THREAD

**Issue by grenlayk** [googleinterns/step259-2020#39]
I have some troubles with 
```
GET https://grenlayk-step-2020.appspot.com/meal/similar?id=2  
```
Only in deployed project  - I receive 500 error, but this thing does not happen if I run project locally. 

@bprusak  @zajonc-gugle Do you have any ideas what can it be?

**Comment by grenlayk** [comment]
All unit tests are passed even in deployment stage. That's why it's really strange.

**Comment by bprusak** [comment]
I found that there's a `DatastoreNeedsIndexException` for these calls: https://pantheon.corp.google.com/errors/CJmkx-u8wJKLEQ?time=PT1H&project=grenlayk-step-2020
According to [the indexes documentation](https://cloud.google.com/datastore/docs/concepts/indexes) only simple indexes exist in Datastore by default, where "simple" means "an index that considers only one field". For a two-field filter we need to define an index explicitly in a `WEB-INF/index.yaml` file.
That's what I did, and I used a `gcloud` command to update the indexes manually. I'll submit the file I used in a PR soon.
